src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+                                    <VisualState x:Name="Normal" />
+                                    <VisualState x:Name="PointerOver">
+                                        <VisualState.Setters>
+                                            <Setter Target="LayoutRoot.Background" Value="{ThemeResource ListMediumControlBackgroundBrush}" />
+                                            <Setter Target="AccessibilityBorder.BorderBrush" Value="{ThemeResource ControlPointerOverBorderBrush}" />
+                                            <Setter Target="AccessibilityBorder.Background" Value="{ThemeResource ControlSelectedBackgroundBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Pressed">
+                                        <VisualState.Setters>
+                                            <Setter Target="LayoutRoot.Background" Value="{ThemeResource BaseMediumLowPressedBackgroundBrush}" />
+                                            <Setter Target="AccessibilityBorder.BorderBrush" Value="{ThemeResource ControlPointerOverBorderBrush}" />
+                                            <Setter Target="AccessibilityBorder.Background" Value="{ThemeResource ControlSelectedBackgroundBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="CheckStates">
+                                    <VisualState x:Name="Checked">
+                                        <VisualState.Setters>
+                                            <Setter Target="VisualBorder.BorderBrush" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="VisualBorder.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ForegroundColorBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Unchecked" />
+                                    <VisualState x:Name="Indeterminate" />
+                                </VisualStateGroup>
+                            </VisualStateManager.VisualStateGroups>
+                        </Grid>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Page.Resources>
+
+    <Grid>
+        <StackPanel Margin="10,10,10,10">
+            <TextBlock x:Name="GridHeading" Text="Grid_d" FontSize="28"/>
```

Comment:
I think we just delete the Text="Grid_d" from here and the rest of the TextBlocks. Also I think assigning the localized strings would be better if we use x:Uid instead. See my other comment in the code that assigns the localize strings for more details.

Reply:
The text assignments are just meant to be design time so that when looking at the control in the editor it isn't just a bunch of blanks.

Reply:
The localized string by way of x:Uid is done.  But still left the design time strings there.